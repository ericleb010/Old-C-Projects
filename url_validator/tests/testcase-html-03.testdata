-d 0 http://qemg.mta.ca/~andrew/comp2931/simplepages/pageA.html
