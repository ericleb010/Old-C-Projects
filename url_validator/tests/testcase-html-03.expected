
Checking http://qemg.mta.ca/~andrew/comp2931/simplepages/pageA.html ... OK
